code/datums/status_effects/debuffs/drunk.dm

```diff
@@ -192,11 +192,13 @@
 /datum/status_effect/inebriated/drunk/proc/attempt_to_blackout()
 	var/mob/living/carbon/drunkard = owner
 	if(drunkard.gain_trauma(/datum/brain_trauma/severe/split_personality/blackout, TRAUMA_LIMIT_ABSOLUTE))
-		drunk_value -= 50 //So that the drunk personality can spice things up without being killed by liver failure
+		drunk_value -= 70 //So that the drunk personality can spice things up without being killed by liver failure
+		return
+	else if(drunkard.has_trauma_type(/datum/brain_trauma/severe/split_personality/blackout))
+		if(prob(10))
+			to_chat(owner, span_warning("You stumbled and fall over!"))
+			owner.slip(1 SECONDS)
```

Comment:
Hey, can you move this check before the gain_trauma line? That may fix #79138, but at the least we won't constantly attempt to give the trauma when the mob already has it. Also remove the `else` from the `if` control statement, it's unnecessary around `return` statements.